src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -408,6 +408,42 @@ public final BeanDescription introspectDirectClassAnnotations(JavaType type) {
     }
 
     /**
+     * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+     * possible per-type override for given property type.<br>
+     * NOTE: if no override found, defaults to value returned by
+     * {@link #getDefaultPropertyInclusion()}.
+     *
+     * @since 2.9
+     */
+    public abstract JsonInclude.Value getDefaultInclusion(Class<?> baseType,
+            Class<?> propertyType);
+
+    /**
+     * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+     * possible per-type override for given property type; but
+     * if none found, returning given <code>defaultIncl</code>
+     *
+     * @param defaultIncl Inclusion setting to return if no overrides found.
+     *
+     * @since 2.9
+     */
+    public JsonInclude.Value getDefaultInclusion(Class<?> baseType,
+            Class<?> propertyType, JsonInclude.Value defaultIncl)
+    {
+        JsonInclude.Value v0 = getConfigOverride(propertyType).getIncludeAsProperty();
+        if (v0 != null) {
+            return v0;
```

Comment:
Yes, I think this is correct. I do not recall why merge wasn't done in the other case; may have been a simple oversight on my part.